modules/placement-driver/src/main/java/org/apache/ignite/internal/placementdriver/PlacementDriverManager.java

```diff
@@ -258,16 +245,6 @@ private void recoverInternalComponentsBusy() {
 
         long recoveryRevision = recoveryFinishedFuture.join();
 
-        CompletableFuture<Void> startLeaserTrackerFuture = leaseTracker.startTrackAsync(recoveryRevision);
-
-        // Forces to wait until recovery is complete before the metastore watches is deployed to avoid races with other components.
-        startVv.update(recoveryRevision, (unused, throwable) -> startLeaserTrackerFuture)
-                .whenComplete((unused, throwable) -> {
-                    if (throwable != null) {
-                        LOG.error("Error starting the PlacementDriverManager internal components", throwable);
-                    } else {
-                        LOG.debug("Internal components of the PlacementDriverManager started successfully");
-                    }
-                });
+        leaseTracker.startTrackAsync(recoveryRevision);
```

Comment:
This change is not obvious to me. Can you please comment on it?
We also need a review from the "class owner"

Reply:
We don't need to fire recovery during recovery, so we don't need to bring back the future anymore.

Reply:
Fix naming